admin/osx/mac-crafter/Sources/Commands/Build.swift

```diff
@@ -1,210 +1,214 @@
-/*
- * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
- * SPDX-License-Identifier: GPL-2.0-or-later
- */
+// SPDX-FileCopyrightText: Nextcloud GmbH
+// SPDX-FileCopyrightText: 2024 Claudio Cambra
+// SPDX-FileCopyrightText: 2025 Iva Horn
+// SPDX-License-Identifier: GPL-2.0-or-later
 
 import ArgumentParser
 import Foundation
 
-struct Build: ParsableCommand {
+struct Build: AsyncParsableCommand {
     static let configuration = CommandConfiguration(abstract: "Client building script")
-
+    
     @Argument(help: "Path to the root directory of the Nextcloud Desktop Client git repository.")
     var repoRootDir = "\(FileManager.default.currentDirectoryPath)/../../.."
-
+    
     @Option(name: [.short, .long], help: "Code signing identity for desktop client and libs.")
     var codeSignIdentity: String?
-
+    
     @Option(name: [.short, .long], help: "Path for build files to be written.")
     var buildPath = "\(FileManager.default.currentDirectoryPath)/build"
-
+    
     @Option(name: [.short, .long], help: "Path for the final product to be put.")
     var productPath = "\(FileManager.default.currentDirectoryPath)/product"
-
+    
     @Option(name: [.short, .long], help: "Architecture.")
     var arch = "arm64"
-
+    
     @Option(name: [.long], help: "Brew installation script URL.")
     var brewInstallShUrl = "https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh"
-
+    
     @Option(name: [.long], help: "CraftMaster Git URL.")
-    var craftMasterGitUrl = "https://invent.kde.org/packaging/craftmaster.git"
-
+    var craftMasterGitUrl = "https://invent.kde.org/ggadinger/craftmaster.git"
```

Comment:
We can use the upstream CraftMaster by default again :)

```suggestion
    var craftMasterGitUrl = "https://invent.kde.org/packaging/craftmaster.git"
```

Reply:
Ah, yes, this pull request did not take your pull request into consideration which was integrated meanwhile. A race condition.